jrnl/plugins/fancy_exporter.py

```diff
@@ -33,16 +36,18 @@ def export_entry(cls, entry):
         """Returns a fancy unicode representation of a single entry."""
         date_str = entry.date.strftime(entry.journal.config["timeformat"])
         linewrap = entry.journal.config["linewrap"] or 78
-        initial_linewrap = linewrap - len(date_str) - 2
+        initial_linewrap = max((1, linewrap - len(date_str) - 2))
```

Comment:
Would it make more sense here to check if `initial_linewrap` becomes negative and throw the error?

Reply:
It'd make sense. I thought about it and decided to go the route I did thinking that we could still squeeze out some narrow linewrap values (I don't have a practical reason for it). 

That said, Simply enforcing that `initial_linewrap` is not negative simplifies the code flow, though I'll need to rework the exception since there is no `card` at that point in the execution. 

I have no strong preference, so give me a direction to go in after looking at the rest of the PR updates.

Reply:
Got it. I don't feel very strongly about this either, and I think it's pretty clear as is right now. Thanks for the explanation.